src/k8s/subscriptions/DEV-CSTAR/terraform.tfvars

```diff
@@ -224,6 +224,7 @@ configmaps_rtdpaymentinstrumentmanager = {
   POSTGRES_POOLSIZE                                 = "2"
   POSTGRES_SHOW_SQL                                 = "true"
   LOG_LEVEL_RTD_PAYMENT_INSTRUMENT_MANAGER          = "INFO"
+  BATCH_PAYM_INSTR_EXTR                             = "0 */15 * * * *"
```

Comment:
@alberto-re-pagopa Not blocking, but for testing I think it's better to have a shorter period, like 5 minutes

Reply:
OK, will schedule at every 5'